docs/reference/settings/ilm-settings.asciidoc

```diff
@@ -0,0 +1,15 @@
+[role="xpack"]
+[[ilm-settings]]
+=== {ilm-cap} settings
+
+These index-level {ilm-init} settings are typically configured through index
+templates. For more information, see <<ilm-gs-create-policy>>.
+
+`index.lifecycle.name`::
+The name of the policy to use to manage the index.
+
+`index.lifecycle.rollover_alias`::
+The index alias to update when the index rolls over. Required if the specified
+policy contains a rollover action. When the index rolls over, the alias is
+updated to reflect that the index is no longer the
+<<aliases-write-index, write index>>.
```

Comment:
the subtlety here is that rolling-over with an explicit write index is not a requirement in the context of ILM. It is just highly recommended. Highly recommended in all cases, even outside of ILM 😄 . For this reason, should we link to the rollover docs here?